pre-docs/inventory_version2.md

```diff
@@ -312,12 +312,13 @@ groups:
 The Terraform plugin supports looking up target object from a Terraform state
 file. It accepts several fields:
 
-`dir`: The directory from which to load Terraform state
+`dir`: The directory containing either a local Terraform state file or Terraform configuration to read remote state from
 `resource_type`: The Terraform resources to match, as a regular expression
 `uri`: The property of the Terraform resource to use as the target URI (optional)
-`statefile`: The name of the Terraform state file to load within `dir` (optional, defaults to `terraform.tfstate`)
+`statefile`: The name of the local Terraform state file to load within `dir` (optional, defaults to `terraform.tfstate`)
```

Comment:
Can we note that this is *always* relative to the `dir` and that absolute paths will not work.